research/src/pages/skeleton.research.mdx

```diff
@@ -0,0 +1,15 @@
+---
+name: Skeleton
+path: /components/skeleton.research
+menu: Analysis
```

Comment:
You should probably add `pathToProposal` here.

Reply:
Why? None of other proposals has it and it's not read anywhere